contracts/extensions/public-pools-strategy-manager.clar

```diff
@@ -34,3 +47,23 @@
 		(ok (map-set authorised-managers who enabled))
 	)
 )
+
+;; Dead Man's switch
+
+(define-read-only (dms-active)
```

Comment:
dms-active sounds like the dms was pressed. Better "dms-inactive"?

Reply:
Perhaps `dms-available`? I called it "active" because it indicates the DMS "can be pressed".

Reply:
Probably, I don't understandthe term DMS not properly. It indicates that the manager was not active, correct? 
Go with whatever you like.